app.rb

```diff
@@ -0,0 +1,26 @@
+require 'sinatra/base'
+require 'pg'
+
+class Chitter < Sinatra::Base
+    enable :sessions
+    
+    get "/" do
+      
+      erb :index
+    end
+  
+    post "/peeps" do
+    session[:peep] = params[:peep]
+   
+    # peep = params['peeps']
+    # connection = PG.connect(dbname: 'chitter_test')
+    # connection.exec("INSERT INTO peeps (peep) VALUES('#{peep}')")
```

Comment:
To try and get these three commented out lines to work, I checked your migrations (very useful to have, thanks!) and saw `CREATE TABLE chitter` - so that made sense of the error page which said there was no relation (table) called peeps. I changed `INSERT INTO peeps` to `INSERT INTO chitter`, at which point I noticed in the database that I was able to create peeps from your website, but they were all empty. So I guessed that there was a problem with `#{peep}` here - it was probably empty. I'll leave it to you to wonder about what went wrong on line 15!